.github/workflows/check-codeowners.yaml

```diff
@@ -3,7 +3,7 @@ name: Check CODEOWNERS
 on: pull_request
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Should we change this to ubuntu-latest? The version really doesn’t matter for this workflow.

Reply:
I would als prefer lastest, but I through there was a reason for pin the version. I didn't know each historic decision

Reply:
i would favor explicit updates instead of using latest.

Reply:
But it's not constant across all workflows

https://github.com/prometheus-community/helm-charts/blob/8995925121ff0548842ab99dc2134659098e8927/.github/workflows/superlinter.yml#L11

Reply:
Than I would prefer to have the fixed version everywhere.

Imho it's not more maintenance as renovate bot should take care of it.

Greetings from kubecon😁

Reply:
Lets get together at kubecon? If you want, write me at cncf slack or linkedin

Reply:
Lets get together at kubecon? If you want, write me at cncf slack or linkedin

Reply:
Will do :)